### PR TITLE
Change course ownership on edit also added warning popup.

### PR DESCRIPTION
**Pull request by waheedahmed**
[ECOM-7415](https://openedx.atlassian.net/browse/ECOM-7415)

As a MKT reviewer or CT reviewer, when I edit a Course and I don't own the Course, the ownership should be transferred to the user making the edit.

**When Course Team Reviewing the Course:**
----------------------------------------------------------------------------------------------------
screen shot 2017-03-20 at 1 55 43 pm

**When Marketing Team Reviewing the Course:**
----------------------------------------------------------------------------------------------------
screen shot 2017-03-20 at 2 05 41 pm

### CODE REVIEW

**Comment by waheedahmed** [comment]
@awais786 @tasawernawaz Please review.

File: course_discovery/apps/publisher/tests/test_views.py [hunk 1]
```diff
@@ -1935,6 +1952,35 @@ def test_update_course_with_state(self):
...
+
+        self.assertRedirects(
+            response,
+            expected_url=reverse('publisher:publisher_course_detail', kwargs={'pk': self.course.id}),
+            status_code=302,
+            target_status_code=200
+        )
+
+        course_state = CourseState.objects.get(id=self.course.course_state.id)
+        self.assertEqual(course_state.name, CourseStateChoices.Draft)
```
Comment (tasawernawaz) [inline course_discovery/apps/publisher/tests/test_views.py:1988]: We have already test for change state on editing https://github.com/edx/course-discovery/pull/761/files#diff-7541c140671314e6639d6a0fe3f4c4dfL1912
You can remove redundancy.

**Review by tasawernawaz — APPROVED** [review]
just a minor comment, otherwise LGTM 👍

File: course_discovery/apps/publisher/tests/test_views.py (hunk 1 above)
Comment (waheedahmed) [reply course_discovery/apps/publisher/tests/test_views.py:1988]: @tasawernawaz But we need to verify that both things happening on editing.

Comment (tasawernawaz) [reply course_discovery/apps/publisher/tests/test_views.py:1988]: 👍

**Review by awais786 — APPROVED** [review]

**Pull request merged**